test/langtools/tools/javac/diags/examples/UnderscoreAsIdentifierError.java

```diff
@@ -22,6 +22,8 @@
  */
 
 // key: compiler.err.underscore.as.identifier
+// key: compiler.warn.source.no.system.modules.path
+// options: -source 20
```

Comment:
Nit: either `-Xlint:-options -source 21`, or `--release 21`, and drop the `warn.source.no.system.modules.path`.